scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/JobEmailNotification.java

```diff
@@ -160,11 +163,17 @@ private String getBody() {
         String jobID = jobState.getId().value();
         String status = jobState.getStatus().toString();
         String hostname = "UNKNOWN";
+        List<TaskState> tasks = jobState.getTasks();
+        StringBuilder taskStatus = new StringBuilder();
+        for (TaskState task : tasks) {
+            taskStatus.append(task.getId().getReadableName() + " (" + task.getId().toString() + ") Status: " +
+                              task.getStatus().toString() + "\n");
+        }
```

Comment:
You could replace this for loop and the StringBuilder by:
`String.join(System.lineSeparator(), tasks.stream().map(task -> task.getId().getReadableName() + " (" + task.getId().toString() + ") Status: " + task.getStatus().toString()).collect(Collectors.toList()));`